libs/model/test/contest/contests-projection-lifecycle.spec.ts

```diff
@@ -161,9 +157,26 @@ describe('Contests projection lifecycle', () => {
   });
 
   it('should project events on multiple contests', async () => {
+    const contestBalance = 10000000000;
```

Comment:
`1_000` syntax would be preferred here

Reply:
good suggestion 👍 ... will do in future PR